### PR TITLE
LPS-64308

### PR DESCRIPTION
**Pull request by robframpton**
/cc @carloslancha

### CODE REVIEW

File: modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/java/com/liferay/portlet/configuration/css/web/internal/portlet/configuration/icon/PortletConfigurationCSSPortletConfigurationIcon.java [hunk 1]
```diff
@@ -126,4 +99,9 @@ public boolean isToolTip() {
 		return false;
 	}
 
+	@Override
+	public boolean isUseDialog() {
+		return true;
+	}
+
 }
```
Comment (duchuyctlk) [inline modules/apps/web-experience/portlet-configuration/portlet-configuration-css-web/src/main/java/com/liferay/portlet/configuration/css/web/internal/portlet/configuration/icon/PortletConfigurationCSSPortletConfigurationIcon.java]: Please add empty line at the end of each source file.

**Comment by carloslancha** [comment]
cc @ealonso

**Comment by brianchandotcom** [comment]
ci:retest

**Comment by brianchandotcom** [comment]
ci:retest

**Comment by brianchandotcom** [comment]
@ealonso please resend on top of https://github.com/ealonso/liferay-portal/pull/880

See last commit about ordering. Thx.

**Pull request closed without merging**